pptx/package.py

```diff
@@ -74,7 +74,7 @@ def next_image_partname(self, ext):
         def first_available_image_idx():
             image_idxs = sorted([
                 part.partname.idx for part in self.iter_parts()
-                if part.partname.startswith('/ppt/media/image')
+                if part.partname.startswith('/ppt/media/image') and part.partname.idx is not None
```

Comment:
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-1-2f1c7cdd4b6a> in <module>()
     14 shapes.placeholders[0].text = 'dummy text'
     15 
---> 16 shapes.add_picture('test_image.png', left=Inches(0.1), top=Inches(1.1), height=Inches(3.4))

/Users/volodymyrkazantsev/WS/python-pptx/pptx/shapes/shapetree.py in add_picture(self, image_file, left, top, width, height)
    147         object.
    148         """
--> 149         image_part, rId = self._slide.get_or_add_image_part(image_file)
    150         pic = self._add_pic_from_image_part(
    151             image_part, rId, left, top, width, height

/Users/volodymyrkazantsev/WS/python-pptx/pptx/parts/slide.py in get_or_add_image_part(self, image_file)
     39         newly created.
     40         """
---> 41         image_part = self._package.get_or_add_image_part(image_file)
     42         rId = self.relate_to(image_part, RT.IMAGE)
     43         return image_part, rId

/Users/volodymyrkazantsev/WS/python-pptx/pptx/package.py in get_or_add_image_part(self, image_file)
     64         otherwise a new one is created.
     65         """
---> 66         return self._image_parts.get_or_add_image_part(image_file)
     67 
     68     def next_image_partname(self, ext):

/Users/volodymyrkazantsev/WS/python-pptx/pptx/package.py in get_or_add_image_part(self, image_file)
    137         image_part = self._find_by_sha1(image.sha1)
    138         if image_part is None:
--> 139             image_part = ImagePart.new(self._package, image)
    140         return image_part
    141 

/Users/volodymyrkazantsev/WS/python-pptx/pptx/parts/image.py in new(cls, package, image)
     40         |Image| object.
     41         """
---> 42         partname = package.next_image_partname(image.ext)
     43         return cls(
     44             partname, image.content_type, image.blob, package, image.filename

/Users/volodymyrkazantsev/WS/python-pptx/pptx/package.py in next_image_partname(self, ext)
     83             return len(image_idxs)+1
     84 
---> 85         idx = first_available_image_idx()
     86         return PackURI('/ppt/media/image%d.%s' % (idx, ext))
     87 

/Users/volodymyrkazantsev/WS/python-pptx/pptx/package.py in first_available_image_idx()
     74         def first_available_image_idx():
     75             image_idxs = sorted([
---> 76                 part.partname.idx for part in self.iter_parts()
     77                 if part.partname.startswith('/ppt/media/image')
     78             ])

TypeError: unorderable types: NoneType() < NoneType()